inc/wpseo-non-ajax-functions.php

```diff
@@ -12,298 +12,86 @@
 }
 
 /**
- * Adds an SEO admin bar menu with several options. If the current user is an admin he can also go straight to several settings menu's from here.
+ * Initializes the admin bar.
+ *
+ * @return void
  */
-function wpseo_admin_bar_menu() {
-	// If the current user can't write posts, this is all of no use, so let's not output an admin menu.
-	if ( ! current_user_can( 'edit_posts' ) ) {
-		return;
-	}
-
-	$options = WPSEO_Options::get_options( array( 'wpseo', 'wpseo_ms' ) );
+function wpseo_initialize_admin_bar() {
+	$admin_bar_menu = new WPSEO_Admin_Bar_Menu();
+	$admin_bar_menu->register_hooks();
+}
+add_action( 'wp_loaded', 'wpseo_initialize_admin_bar' );
 
-	if ( $options['enable_admin_bar_menu'] !== true ) {
-		return;
+/**
+ * Allows editing of the meta fields through weblog editors like Marsedit.
+ *
+ * @param array $allcaps Capabilities that must all be true to allow action.
+ * @param array $cap     Array of capabilities to be checked, unused here.
+ * @param array $args    List of arguments for the specific cap to be checked.
+ *
+ * @return array $allcaps
+ */
+function allow_custom_field_edits( $allcaps, $cap, $args ) {
```

Comment:
I am not a big fan of abbreviations. I would choose for  `$capabilities` and `$arguments` (or something else because arguments is not very descriptive)

Reply:
I generally agree, but I think here we should keep it that way, as those abbreviated terms are common in WordPress code.